packages/paste-core/components/table/src/Tr.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {TrProps, TrPropTypes} from './types';
+
+const Tr = React.forwardRef<HTMLTableRowElement, TrProps>(({verticalAlignment = 'middle', ...props}, ref) => {
+  return (
+    <Box
+      {...safelySpreadBoxProps(props)}
+      ref={ref}
+      as="tr"
+      borderStyle="solid"
+      borderColor="colorBorderLighter"
+      borderWidth="borderWidth0"
+      borderBottomWidth="borderWidth10"
+      verticalAlign={verticalAlignment}
+      _even={{backgroundColor: 'colorBackgroundRowStriped'}}
```

Comment:
We could potentially deprecate this token, and use colorBackground here instead.

@SiTaggart originally added this token, so will wait for him to chime in.